client/components/dialog/dialog-base.jsx

```diff
@@ -29,6 +30,16 @@ var DialogBase = React.createClass( {
 		};
 	},
 
+	componentWillMount() {
+		this.props.setOnEscCallback( this._close );
+	},
+
+	componentWillReceiveProps(nextProps) {
+		if (nextProps.isEscPressed) {
```

Comment:
Need spaces between parens ([reference](https://github.com/Automattic/wp-calypso/blob/2490b4cb4f22ee87fd050e14124137cd42d4d381/docs/coding-guidelines/javascript.md#examples-of-good-spacing)).